src/cmd-build

```diff
@@ -75,6 +77,12 @@ while true; do
         --delay-meta-merge)
             DELAY_META_MERGE=true
             ;;
+        --config-archive)
+            CONFIG_ARCHIVE=true
+            ;;
```

Comment:
I don't understand the value of an explicit option to set the default.  If we were to do this it'd make more sense to have `--config-archive=false` type args right?

Reply:
I guess the convention used here would maybe be to have `--skip-config-archive` instead of `--no-config-archive`.  None of the binary state command-line options do it with arguments currently, and evaluating the argument of a binary state option adds complexity and overhead to allow for the variation (or not) in the value passed.

I can change the variable to be `--skip-config-archive` instead of `--no-config-archive`.